ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -178,6 +179,13 @@ RnsShell::LayerInvalidateMask RSkComponentImage::updateComponentProps(const Shad
       imageProps.tintColor = RSkColorFromSharedColor(newimageProps.tintColor,SK_ColorTRANSPARENT);
     }
     if((forceUpdate) || (oldimageProps.sources[0].uri.compare(newimageProps.sources[0].uri) != 0)) {
+      if( isRequestInProgress_ && remoteCurlRequest_){
+        auto sharedCurlNetworking = CurlNetworking::sharedCurlNetworking();
+        // if url is changed, image component is get component property update.
+        // calncel the onging request and made new request to network.  
+        sharedCurlNetworking->abortRequest(remoteCurlRequest_);
```

Comment:
whether onLoadEnd event need to be sent here ?
maybe we can add a comment here about it.